src/utils.ts

```diff
@@ -85,7 +85,9 @@ export function getWalletWithProvider({
 
   const wallet = new Wallet(privateKey);
   let provider: providers.BaseProvider = new providers.JsonRpcProvider(
-    chain === "local-tableland" ? undefined : providerUrl, // Defaults to localhost
+    chain === "local-tableland" && !providerUrl
+      ? "http://127.0.0.1:8545"
+      : providerUrl,
     network.name === "localhost" ? undefined : network.name
```

Comment:
Are we sure _this_ isn't the line that is causing @dtbuchholz's issue?

Reply:
I was able to reproduce the issue and these changes got things working for me.  @dtbuchholz Can you double check?

Reply:
Yup, can confirm it resolved the issue.

Reply:
:shipit: